### PR TITLE
docs: Make clearer it's Javascript, not Java

### PR DESCRIPTION
**Pull request by cobraz**
Oh, man! Thanks for this awesome provider 🎉

This pull request adds a hyphen `-` to make it clearer it's for Javascript, not Java.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ To use this package, please [install the Pulumi CLI first](https://pulumi.io/).
 
 This package is available in many languages in the standard packaging formats.
 
-### Node.js (Java/TypeScript)
+### Node.js (Java-/TypeScript)
```
Comment (mikhailshilkov) [inline README.md:15]:
```suggestion
### Node.js (JavaScript/TypeScript)
```

I guess we don't have to save keystrokes here :)

**Review by mikhailshilkov — COMMENTED** [review]
Thank you for your attention to detail!

Could you please update [this file](https://github.com/pulumi/pulumi-google-native/blob/master/sdk/python/README.md) too? It's just a copy of the same README.

I also see that we have the same issue in almost every other Pulumi provider...

File: README.md (hunk 1 above)
Comment (cobraz) [reply README.md:15]: 👍

**Comment by cobraz** [comment]
> I also see that we have the same issue in almost every other Pulumi provider...

I can see if I'm able to create a simple script that adds a bunch of PRs 😅

**Comment by mikhailshilkov** [comment]
> I can see if I'm able to create a simple script that adds a bunch of PRs 😅

@stack72 Do we generate READMEs in TF-based providers or are those manually edited?

**Review by mikhailshilkov — APPROVED** [review]

**Comment by stack72** [comment]
The READMEs are manually written but they are also present in sdk/python as well in each of the providers

**Pull request merged**